packages/server/src/course/course.controller.ts

```diff
@@ -133,14 +134,21 @@ export class CourseController {
     );
     let all: AsyncQuestionModel[] = [];
 
-    if (Object.keys(asyncQuestionKeys).length === 0) {
+    if (true) {
```

Comment:
Might want to change this back to using redis

```suggestion
    if (Object.keys(asyncQuestionKeys).length === 0) {
```